src/popup/router/pages/Names/Claim.vue

```diff
@@ -1,34 +1,59 @@
 <template>
   <div class="claim">
-    <div class="claim-name-holder">
-      <InputField
-        v-model="name"
-        :placeholder="$t('pages.names.claim.name-placeholder')"
-        :error="!validName"
-      >
-        <span slot="right">.chain</span>
-      </InputField>
-      <Button
-        small
-        :disabled="!sdk || !validName"
-        @click="claim"
-      >
-        <ae-icon name="plus" />
-      </Button>
+    <span>
+      {{ $t('pages.names.claim.register-name') }}
+    </span>
+    <InputField
+      v-model="name"
+      :placeholder="$t('pages.names.claim.name-placeholder')"
+      :error="!!name.length && !validName"
+    >
+      <span slot="right">.chain</span>
+    </InputField>
+    <div class="flex-row">
+      <CheckBox v-model="autoExtend">
+        {{ $t('pages.names.claim.auto-extend') }}
+        <HintButton
```

Comment:
Should be blue as well.

![Screenshot 2021-07-22 at 17 02 13](https://user-images.githubusercontent.com/482351/126652078-862e17f7-539b-4723-a2be-e549d8adc982.png)

Reply:
They've changed after this PR was made